### PR TITLE
Log levels as DEBUG, INFO, WARN and ERROR and new proof of life logging message

### PR DESCRIPTION
**Pull request by gtorodelvalle**
- 100% tests passed
- Implements https://github.com/telefonicaid/IoT-STH/issues/74
- Assigned to @frbattid 

Paco, due to the new https://github.com/telefonicaid/IoT-STH/blob/develop/CHANGES_NEXT_RELEASE file, the PRs will probably need me to rebase them before landing... I'll rebase them as soon as I notice other land ;) Just for you to know :)

### CODE REVIEW

File: src/sth_logger.js [hunk 1]
```diff
@@ -11,21 +11,17 @@
   var sthConfig;
 
   var LOG_LEVEL = {
-    trace: 'trace',
-    info: 'info',
     debug: 'debug',
+    info: 'info',
     warn: 'warn',
-    error: 'error',
-    fatal: 'fatal'
```
Comment (frbattid) [inline src/sth_logger.js]: Just curiosity, why `fatal` level has disappeared?

Comment (gtorodelvalle) [inline src/sth_logger.js]: @mrutid mentioned me that other components were using only these 4... :)

Comment (frbattid) [inline src/sth_logger.js]: Umm, Cygnus uses the `fatal` one xD

I'll check it later, in the meantime, NTC

**Comment by frbattid** [comment]
LGTM

**Pull request merged**